### PR TITLE
Feature/grails5

### PR DESCRIPTION
**Pull request by sughics**
PR for https://github.com/AtlasOfLivingAustralia/collectory/issues/181

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -100,107 +78,106 @@ sourceSets {
...
+    testImplementation "org.seleniumhq.selenium:selenium-api:$seleniumVersion"
+    testImplementation "org.seleniumhq.selenium:selenium-support:$seleniumVersion"
+    runtimeOnly "net.sourceforge.htmlunit:htmlunit:2.18"
+    testImplementation "org.seleniumhq.selenium:selenium-htmlunit-driver:2.47.1"
+    testImplementation "org.seleniumhq.selenium:selenium-chrome-driver:$seleniumVersion"
+    testImplementation "org.seleniumhq.selenium:selenium-firefox-driver:$seleniumVersion"
+    testImplementation "org.seleniumhq.selenium:selenium-safari-driver:$seleniumSafariDriverVersion"
+    implementation 'org.grails.plugins:sentry:11.7.25'
+    // db-migration
+    implementation 'org.liquibase:liquibase-core:4.19.0'
```
Comment (vjrj) [inline build.gradle:144]: Thinking in the grails migration, I was worried about this liquibase part, bit I see you did it great. But in spatial-service for some reason the liquibase support was removed https://github.com/AtlasOfLivingAustralia/spatial-service/issues/213

Comment (sughics) [reply build.gradle:144]: @adam-collins Re: spatial liquibase.

**Review by vjrj — APPROVED** [review]
LGTM. The only concern is that the CollectoryAuthService only works with oidc but not with `security.cas.enabled` as before.

image

**Comment by djtfmartin** [comment]
> LGTM. The only concern is that the CollectoryAuthService only works with oidc but not with `security.cas.enabled` as before.
> 

@sughics is this intentional / expected ?

**Comment by sughics** [comment]
@vjrj , the issue you raised with `security.cas.enabled` has now been resolved with a new ala-security-project version i.e. 6.0.4 and 6.1.0-SNAPSHOT.
Please ensure that if you are using non oidc CAS deployment that these config values are set to false. 
`security.cas.enabled`, `security.jwt.enabled`, and `webservice.jwt`

**Pull request merged**